files/system/usr/share/ublue-os/just/60-custom.just

```diff
@@ -489,10 +489,13 @@ audit-secureblue:
     fi
 
     EXTENSIONS_TEST_STRING="Ensuring GNOME user extensions are disabled"
-    if [ "$(gsettings get org.gnome.shell allow-extension-installation)" = "false" ]; then
-        print_status "$EXTENSIONS_TEST_STRING" "$STATUS_SUCCESS"
-    else
-        print_status "$EXTENSIONS_TEST_STRING" "$STATUS_FAILURE"
+    GSETTINGS_LOCATION="/usr/bin/gsettings"
```

Comment:
this should probably be:

```
if command -v gsettings &> /dev/null; then
```

since it's not sufficient that the gsettings bin exists, if it's not on the $PATH

Reply:
suggestion tested and incorporated